src/error.rs

```diff
@@ -1116,7 +1116,10 @@ impl ToDiagnostic<FileId> for EvalError {
 
                 vec![Diagnostic::error()
                     .with_message("non mergeable terms")
-                    .with_labels(labels)]
+                    .with_labels(labels)
+                    .with_notes(vec![String::from(
+                        "Both values have the same merge priority but they can't be combined",
```

Comment:
```suggestion
                        "Both values have the same merge priority so they can't be combined",
```

Reply:
I would keep the `but`. 2 values with the same priority may very well be combined, like records. Here the problem is that they have the same priority AND we don't know how to merge them.